CONTRIBUTING.md

```diff
@@ -555,6 +555,31 @@ options we have on `Aer` to CMake we use it's native mechanism:
     (QiskitDevEnv) qiskit-aer\out> cmake -G "Visual Studio 15 2017" -DBLAS_LIB_PATH=c:\path\to\my\blas ..
 
 
+### Building with GPU support
+
+Qiskit Aer can exploit GPU's horsepower to accelerate some simulations, specially the larger ones.
+GPU access is supported via CUDA® (NVIDIA® chipset), so in order to build with GPU support we need
+to have CUDA® >= 10.1 preinstalled. See install instructions [here](https://developer.nvidia.com/cuda-toolkit-archive)
+Please note that we only support GPU acceleration on Linux platforms at the moment.
+
+Once CUDA® is properly installed, we only need to set a flag so the build system knows what to do:
+
+```
+AER_THRUST_BACKEND=CUDA
+```
+
+For example,
+
+    qiskit-aer$ python ./setup.py bdist_wheel -- -DAER_THRUST_BACKEND=CUDA
+
+
+Few notes on GPU builds:
+1. Building takes considerable more time than non-GPU build, so be patient :)
+2. CUDA® >= 10.1 imposes the restriction of building with g++ version not newer than 8
```

Comment:
CUDA 10.1 requires g++ <= 8, but I think if you have CUDA 10.2 you can use g++ 9 right?

Reply:
8 unfortunately :(